windows/msvce-config.json

```diff
@@ -240,6 +240,11 @@
       },
       "scylladb-seastar": null,
       "agner-fog-vcl": null,
+      "xerces-c": {
```

Comment:
These changes unfortunately won't be picked up by MSVC, but it's ok to leave it I guess